src/shared/GlobalTopBanners/TrialBanner/TrialBanner.tsx

```diff
@@ -72,18 +70,17 @@ const TrialBanner: React.FC = () => {
     return null
   }
 
-  // user is not on a free plan, trial is currently on going
```

Comment:
comment was misleading, technically its checking for if the user is on the trial plan specifically, rather than just "not on a free plan"